src/cbc_sdk/platform/models/base_alert.yaml

```diff
@@ -10,7 +10,7 @@ properties:
     - MINOR
     - SERIOUS
     - CRITICAL
-  create_time:
+  detection_timestamp:
```

Comment:
Are the new fields coming in a future commit?

Reply:
yes